Release/QuickChat/docs/README.md

```diff
@@ -0,0 +1,29 @@
+# How to use the quickChat Module in your Zepeto Project ?
+
+The QuickChat module is the is used for sending in game predefined messages in a Zepeto Multiplayer world. 
+
+In this documentation, you will learn how to add a QuickChat module in your existing zepeto Project. If you are just getting started as a developer in Zepeto, we suggest you to  check out our [beginner guide](https://docs.zepeto.me/studio-item/lang-ko/docs) first. 
```

Comment:
Hmm, is this the correct documentation link? Also please make sure that documentation is the global version without the lang-ko in the url.

Reply:
Yeah, I just noticed that I ctrl+c and ctrl+Ved the wrong link. Thanks for noticing ^^